nomad/encrypter.go

```diff
@@ -61,10 +61,10 @@ func encrypterFromKeystore(keystoreDirectory string) (*Encrypter, error) {
 		if path != keystoreDirectory && info.IsDir() {
 			return filepath.SkipDir
 		}
-		if filepath.Ext(path) != ".json" {
+		if !strings.HasSuffix(path, ".nks.json") {
 			return nil
 		}
-		id := strings.TrimSuffix(filepath.Base(path), ".json")
+		id := strings.TrimSuffix(filepath.Base(path), ".nks.json")
```

Comment:
One last thought—should this be a const? I realize I was painting a shed earlier and missed that question.